upgrade_test.py

```diff
@@ -415,186 +415,205 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
+                self.db_cluster.node_to_upgrade.check_node_health()
+
+                # wait for the prepare write workload to finish
+                self.verify_stress_thread(prepare_write_cs_thread_pool)
+
+                # read workload (cl=QUORUM)
+                self.log.info('Starting c-s read workload (cl=QUORUM n=10000000)')
+                stress_cmd_read_cl_quorum = self.params.get('stress_cmd_read_cl_quorum')
+                read_stress_queue = self.run_stress_thread(stress_cmd=stress_cmd_read_cl_quorum)
+                # wait for the read workload to finish
+                self.verify_stress_thread(read_stress_queue)
+                self.fill_and_verify_db_data('after upgraded one node')
+
+                # read workload
+                self.log.info('Starting c-s read workload for 10m')
+                stress_cmd_read_10m = self.params.get('stress_cmd_read_10m')
+                read_10m_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_read_10m)
+
+                self.log.info('Sleeping for 60s to let cassandra-stress start before the upgrade...')
+                time.sleep(60)
+
+            with self.subTest('Step2 - Upgrade Second Node '):
+                # upgrade second node
+                self.db_cluster.node_to_upgrade = self.db_cluster.nodes[indexes[1]]
+                self.log.info('Upgrade Node %s begin', self.db_cluster.node_to_upgrade.name)
+                self.upgrade_node(self.db_cluster.node_to_upgrade)
+                self.log.info('Upgrade Node %s ended', self.db_cluster.node_to_upgrade.name)
+                self.db_cluster.node_to_upgrade.check_node_health()
+
+                # wait for the 10m read workload to finish
+                self.verify_stress_thread(read_10m_cs_thread_pool)
+                self.fill_and_verify_db_data('after upgraded two nodes')
+
+                # read workload (80m)
+                self.log.info('Starting c-s read workload for 80m')
+                stress_cmd_read_80m = self.params.get('stress_cmd_read_80m')
+                read_80m_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_read_80m)
+                self.log.info('Sleeping for 60s to let cassandra-stress start before the rollback...')
+                time.sleep(60)
+
+            with self.subTest('Step3 - Rollback Second Node '):
+                # rollback second node
+                self.log.info('Rollback Node %s begin', self.db_cluster.nodes[indexes[1]].name)
+                self.rollback_node(self.db_cluster.nodes[indexes[1]])
+                self.log.info('Rollback Node %s ended', self.db_cluster.nodes[indexes[1]].name)
+                self.db_cluster.nodes[indexes[1]].check_node_health()
+
+        with self.subTest('Step4 - Verify data during mixed cluster mode '):
+            self.fill_and_verify_db_data('after rollback the second node')
+
+        # TODO: We may need to filter here some errors as well, but for now limit filtering as much as possible
```

Comment:
It's not enough.

In this job : https://jenkins.scylladb.com/job/scylla-staging/job/amos/job/rolling-upgrade-2019.1/job/rolling-upgrade-centos7/17/execution/node/68/log/

'Failed to load schema' error also raised in re-upgrading the node which were rollbacked.

```
< t:2019-12-27 06:00:45,960 f:upgrade_test.py l:472  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-1 begin
< t:2019-12-27 06:00:45,961 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
... <Failed to load schema> ERR  ## first one

< t:2019-12-27 06:05:13,950 f:upgrade_test.py l:474  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-1 ended
... <Failed to load schema> ERR

< t:2019-12-27 06:09:30,989 f:upgrade_test.py l:498  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-4 begin
< t:2019-12-27 06:09:30,990 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-27 06:15:20,868 f:upgrade_test.py l:500  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-4 ended
< t:2019-12-27 06:18:41,282 f:upgrade_test.py l:515  c:sdcm.tester          p:INFO  > Rollback Node rolling-upgrade-upgrade--  centos-db-node-2462f6d0-0-4 begin
< t:2019-12-27 06:18:41,282 f:upgrade_test.py l:207  c:sdcm.tester          p:INFO  > Rollbacking a Node
< t:2019-12-27 06:20:49,838 f:upgrade_test.py l:517  c:sdcm.tester          p:INFO  > Rollback Node rolling-upgrade-upgrade--  centos-db-node-2462f6d0-0-4 ended
< t:2019-12-27 06:23:46,438 f:upgrade_test.py l:524  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-4 begin
< t:2019-12-27 06:23:46,438 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-27 06:27:30,337 f:upgrade_test.py l:526  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-4 ended
... <Failed to load schema> ERR   ## last one
< t:2019-12-27 06:30:28,294 f:upgrade_test.py l:524  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-3 begin
< t:2019-12-27 06:30:28,295 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-27 06:36:13,655 f:upgrade_test.py l:526  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-3 ended
< t:2019-12-27 06:39:13,322 f:upgrade_test.py l:524  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-2 begin
< t:2019-12-27 06:39:13,322 f:upgrade_test.py l:111  c:sdcm.tester          p:INFO  > Upgrading a Node
< t:2019-12-27 06:45:01,047 f:upgrade_test.py l:526  c:sdcm.tester          p:INFO  > Upgrade Node rolling-upgrade-upgrade--   centos-db-node-2462f6d0-0-2 ended
```

Reply:
If it's expected, I can add it as well.